src/app/components/ItemFilters/ItemFilters.jsx

```diff
@@ -40,24 +40,28 @@ const ItemFilters = (
   useEffect(() => {
     let timeout;
     setSelectedFieldDisplayStr(parsedFilterSelections());
-    // Once the new items are fetched, focus on the filter UI and the
-    // results, but don't do this if the user requested to view all items
-    // until all the items are fetched. It is annoying if the text keeps
-    // getting focused and the page keeps jumping around.
-    if (showAll && finishedLoadingItems) {
-      resultsRef.current && resultsRef.current.focus();
-    } else {
-      // When filtering, delay the focus slightly because
-      // of the loading animation screen.
-      timeout = setTimeout(() => {
+    // Once the new items are filtered and fetched, focus on the filter UI
+    // and the results, but don't do this if the user requested to view all
+    // items until all the items are fetched. It is annoying if the text
+    // keeps getting focused and the page keeps jumping around.
+    // The easiest way to figure out if a filter or year search was done,
+    // is to check if the `selectedFieldDisplayStr` string is not empty.
+    if (selectedFieldDisplayStr) {
```

Comment:
Should this be checking the string's length? I believe this will always return true if it's initialized to an empty string.